packages/web-react/src/components/Dropdown/README.md

```diff
@@ -39,7 +40,94 @@ import { Dropdown } from '@lmc-eu/spirit-web-react/components';
 | `trigger['aria-controls']` | `string`                                           | Trigger aria controls   |
 | `trigger.ref`              | `LegacyRef<HTMLButtonElement & HTMLAnchorElement>` | Trigger reference       |
 
+---
+
+# DropdownModern
+
+⚠️ `DropdownModern` component will be deprecated and renamed to `Dropdown` in the next major version.
+This is the React implementation of the [DropdownModern] component.
```

Comment:
there is no DropdownModern page in any other package, so the link doesn't exist, am I right?

Reply:
link removed